Tasks/DockerV2/dockerlifecycle.ts

```diff
@@ -11,33 +11,30 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     let containerName = tl.getInput("container", true);
     let commandArguments = dockerCommandUtils.getCommandArguments(tl.getInput("arguments", false));
 
+    // Map between specified container names and their actual ID
+    // Allow custom containers names through
+    let container = containerName;
     let containerMap = tl.getVariable("agent.containermapping");
-    if (!containerMap) {
-        throw new Error(tl.loc('ContainerNotFound', containerName));
-    }
-
-    let map: any;
-    try {
-        map = JSON.parse(containerMap);
-    } catch (ex) {
-        throw new Error(tl.loc('ContainerNotFound', containerName));
-    }
-
-    if (!map[containerName]) {
-        throw new Error(tl.loc('ContainerNotFound', containerName));
-    }
-    let containerId = map[containerName].id;
-    if (!containerId) {
-        throw new Error(tl.loc('ContainerNotFound', containerName));
```

Comment:
Do we not want to have errors any more?

Reply:
No. We added a feature that allows customers to specify a container's YAML name and have it map to the ID for starting/stopping. However, customers before this were using start/stop as a raw command. A customer could use docker commands to bring up a container and then want to be able to stop it. Since that container isn't registered with the agent, it won't be in this map.

We need to allow customers to still start/stop containers that the agent doesn't know about,

Reply:
Nit: if they are doing that, we may want to at least log something.

Reply:
Sure